### ISSUE TITLE
Weights & Biases experiments

### ISSUE THREAD

**Issue by ramanshsharma2806** [wri-dssg-omdena/policy-data-analyzer#85]
## Setup for Weights & Biases for our notebook and further experiments

We can use wandb for hyperparameter tuning and most importantly keeping track of experiments. With W&B free hosted service we can set this up efficiently.

Since the team version of wandb is paid (30 day free trial is there however), we will use the following project

### [https://wandb.ai/ramanshsharma/WRI](https://wandb.ai/ramanshsharma/WRI)

Please find the API key to write to the public project in Slack.

## Goals

- [x] Create a shared project on weights & biases for the team to work on.
- [x]  Set up training and validation accuracy/loss, weighted/macro F1 score plotting
- [x]  Set up automatic hyperparameter tuning

Helpful links
1. [Intro to Weights & Biases](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/intro/Intro_to_Weights_%26_Biases.ipynb#scrollTo=wRsHBj-EpDHd)
2. [Official examples](https://github.com/wandb/examples)
3. [Organizing Hyperparameter Sweeps in PyTorch with W&B](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Organizing_Hyperparameter_Sweeps_in_PyTorch_with_W%26B.ipynb)

**Comment by ramanshsharma2806** [comment]
The GitHub branch is
https://github.com/wri-dssg/policy-data-analyzer/tree/wandb-experiments

**Comment by ramanshsharma2806** [comment]
W&B experiments finished, pull request created.
